crates/host/src/wasmbus/mod.rs

```diff
@@ -948,11 +944,13 @@ impl Host {
             "version": env!("CARGO_PKG_VERSION"),
         });
 
-        let nats = async_nats::connect(url.as_str())
+        let nats_url = format!("{}:{}", config.nats_host, config.nats_port);
```

Comment:
We already had a rich type here, which was known to be valid, why would we introduce a worse approach here?